frontend/src/app/submitted-topic-drafts/submitted-topic-draft-action-button/submitted-topic-draft-action-button.component.css

```diff
@@ -0,0 +1,42 @@
+li:hover > ul, .fadering {
```

Comment:
Ist wirklich  die datei notwendig? Die wird doch gar nicht benutzt. Lieber in ein extra branch packen oder so. :P

Reply:
Das debug ich noch. Bitte um Geduld.